hal/src/main/native/athena/FRCDriverStation.cpp

```diff
@@ -302,7 +302,7 @@ char* HAL_GetJoystickName(int32_t joystickNum) {
   } else {
     size_t len = std::strlen(joystickDesc.name);
     char* name = static_cast<char*>(std::malloc(len + 1));
-    std::strncpy(name, joystickDesc.name, len);
+    std::memcpy(name, joystickDesc.name, len);
```

Comment:
Shouldn't this be `len + 1` since `len` is from `strlen`?

Reply:
i don't believe it to be an issue as we are still null terminating the output string right after.

Reply:
But yes, it would have been more proper to do len + 1.

Reply:
Ah yes, good point. I should stop reviewing code on my phone 😅